src/electron/platform/android/scan-results-to-unified-results.ts

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { InstanceResultStatus, UnifiedResult } from 'common/types/store-data/unified-data-interface';
+import { UUIDGeneratorType } from 'common/uid-generator';
+import { RuleInformation } from './rule-information';
+import { RuleInformationProvider } from './rule-information-provider';
+import { RuleResultsData, ScanResults } from './scan-results';
+
+export type ConvertScanResultsToUnifiedResultsDelegate = (scanResults: ScanResults, uuidGenerator: UUIDGeneratorType) => UnifiedResult[];
+
+export function convertScanResultsToUnifiedResults(scanResults: ScanResults, uuidGenerator: UUIDGeneratorType): UnifiedResult[] {
+    if (scanResults == null || scanResults.ruleResults == null) {
+        return [];
+    }
+
+    return createUnifiedResultsFromScanResults(scanResults.ruleResults, uuidGenerator);
+}
+
+function createUnifiedResultsFromScanResults(ruleResults: RuleResultsData[], uuidGenerator: UUIDGeneratorType): UnifiedResult[] {
+    const ruleInformationProvider: RuleInformationProvider = new RuleInformationProvider();
+
+    const unifiedResults: UnifiedResult[] = [];
+
+    for (const ruleResult of ruleResults) {
+        const ruleInformation: RuleInformation = ruleInformationProvider.getRuleInformation(ruleResult.ruleId);
+
+        if (ruleInformation != null) {
+            unifiedResults.push(createUnifiedResult(ruleInformation, ruleResult, uuidGenerator));
+        }
+    }
+
+    return unifiedResults;
+}
+
+function createUnifiedResult(
+    ruleInformation: RuleInformation,
+    ruleResult: RuleResultsData,
+    uuidGenerator: UUIDGeneratorType,
+): UnifiedResult {
+    return {
+        uid: uuidGenerator(),
+        ruleId: ruleInformation.ruleId,
+        status: getStatus(ruleResult.status),
+        identifiers: null,
+        descriptors: null,
+        resolution: {
+            'how-to-fix': ruleInformation.howToFix(ruleResult),
+        },
+    };
+}
+
+function getStatus(status: string): InstanceResultStatus {
+    switch (status) {
+        case 'PASS':
+            return 'pass';
+        case 'FAIL':
+            return 'fail';
+        default:
+            return 'unknown';
```

Comment:
The unit tests give us a [table ](https://dev.azure.com/ms/accessibility-insights-web/_build/results?buildId=39457&view=logs&jobId=3613c2bd-8b62-5a70-8491-3e8459586450&taskId=d1343856-9cd9-5649-cdc6-81bc16053e6f&lineStart=1376&lineEnd=1376&colStart=68&colEnd=127) of code coverage & uncovered lines; you might explore these to see that any we're missing are intentionally skipped. For instance, this line is missed

Reply:
Added a case for this